builtin/builtin.mbti

```diff
@@ -5,6 +5,12 @@ fn abort[T](String) -> T
 
 fn assert_eq[T : Eq + Show](T, T, ~loc : SourceLoc = _) -> Unit!Failure
 
+fn assert_false(Bool, ~loc : SourceLoc = _) -> Unit!Failure
+
+fn assert_ne[T : Eq + Show](T, T, ~loc : SourceLoc = _) -> Unit!Failure
+
```

Comment:
assert_not_eq?

Reply:
we don't care about the error type, can we make it a bit more simpler to use, i.e,

fn assert_ne[T : Eq + Show](T, T, ~loc : SourceLoc = _) -> Unit!